main.py

```diff
@@ -112,16 +109,18 @@ def main():
 def header():
     return flask.render_template('landheader.html')
 
-# landfooter.html
-@app.route('/footer')
-def footer():
-    return flask.render_template('landfooter.html')
-
-# homeheader.html
+#homeheader.html
 @app.route('/homeheader')
 def homeheader():
+    if request.method == 'POST':
```

Comment:
it wont let me remove this in this PR for some reason but these two lines are no longer needed